source/manual/emergency-publishing.html.md

```diff
@@ -50,15 +58,75 @@ The data for the emergency banner is stored in Redis. Jenkins is used to set the
 
 1. Click `Build with Parameters`.
 
-1. Fill in the appropriate variables using the form presented by Jenkins. Use a relative URL (i.e. without the www.gov.uk prefix) if a link is required.
+1. Fill in the appropriate variables using the form presented by Jenkins.
 
 1. Click `Build`.
 
 ![Jenkins Deploy Emergency Banner](images/emergency_publishing/deploy_emergency_banner_job.png)
 
-> **Note**
->
-> The Jenkins job will also clear the Varnish caches and the CDN cache for [a predefined list of 10 URLs](https://github.com/alphagov/govuk-puppet/blob/8cca9aad2b68d6cb396a135f47524fafeca1c947/modules/govuk_jenkins/templates/jobs/clear_cdn_cache.yaml.erb#L22-L34) (including the website root).
+#### EC2: Manually running the rake task
+
+1. SSH into a `frontend` machine in the environment you are deploying the banner on:
+
+    ```bash
+    gds govuk connect -e staging ssh frontend
+    ```
+
+1. Change into the directory for `static`:
+
+    ```bash
+    cd /var/apps/static
+    ```
+
+1. Set banner content as environment variables:
+
+    ```bash
+    CAMPAIGN_CLASS="notable-death|national-emergency|local-emergency"
+    HEADING="replace with heading"
+    SHORT_DESCRIPTION="replace with desciption"
+    LINK="replace with link"
+    LINK_TEXT="replace with link text"
+    ```
+
+    > **Note**
+    >
+    > You may need to escape certain characters (including `,` and `"`) with a
+    > backslash. For example `\,` or `\"`.
+
+1. Run the rake task
+
+    ```bash
+    sudo -u deploy govuk_setenv static bundle exec rake emergency_banner:deploy[$CAMPAIGN_CLASS,$HEADING,$SHORT_DESCRIPTION,$LINK,$LINK_TEXT]
+    ```
+
+#### EKS: Run the rake task
+
+1. Set banner content as environment variables
+
+    ```bash
+    CAMPAIGN_CLASS="notable-death|national-emergency|local-emergency"
+    HEADING="replace with heading"
+    SHORT_DESCRIPTION="replace with desciption"
+    LINK="replace with link"
+    LINK_TEXT="replace with link text"
+    ```
+
+    > **Note**
+    >
+    > You may need to escape certain characters (including `,` and `"`) with a
+    > backslash. For example `\,` or `\"`.
+
+1. Find the name of a pod running Static
+
+    ```bash
+    POD_NAME=$(kubectl -n apps get pods -l=app=static -o go-template --template '{{ (index .items 0).metadata.name }}')
+    ```
+
+1. Run the rake task in the app container
+
+    ```bash
+    kubectl -n apps exec -i -t $POD_NAME -- rake "emergency_banner:deploy[$CAMPAIGN_CLASS,$HEADING,$SHORT_DESCRIPTION,$LINK,$LINK_TEXT]"
```

Comment:
We can simplify these two steps to `kubectl exec -it deploy/static -- rake "emergency_banner:...`.